ucc/compile.py

```diff
@@ -49,7 +48,10 @@ def compile(
 
     # Translate to Qiskit Circuit object
     qiskit_circuit = transpile(circuit, "qiskit")
-    compiled_circuit = UCCDefault1(target_device=target_device).run(
+    ucc_default1 = UCCDefault1(target_device=target_device)
+    if custom_passes is not None:
+        ucc_default1.pass_manager.append(custom_passes)
+    compiled_circuit = ucc_default1.run(
```

Comment:
Should the variable still be named `ucc_default1` if it gets modified and new passes get appended?

Reply:
No strong preference; it is still an instance of that class even if we are customizing it. Open to another (shorter =) ) name if you have one.

Reply:
Nothing comes to mind, and it's not blocking from me!